test/ejabberd_tests/tests/mam_SUITE.erl

```diff
@@ -1837,10 +1877,8 @@ parse_error_iq(#xmlel{name = <<"iq">>,
 generate_rpc_jid({_,User}) ->
     {username, Username} = lists:keyfind(username, 1, User),
     {server, Server} = lists:keyfind(server, 1, User),
-    %% esl-ejabberd uses different record to store jids
-     %JID = <<Username/binary, "@", Server/binary, "/rpc">>,
-     %{jid, JID, Username, Server, <<"rpc">>}.
-    {jid, Username, Server, <<"rpc">>, Username, Server, <<"rpc">>}.
+    JID = <<Username/binary, "@", Server/binary, "/rpc">>,
+    escalus_ejabberd:rpc(jlib, binary_to_jid, [JID]).
```

Comment:
I can change it, but it would also require modification here:
https://github.com/esl/MongooseIM/blob/master/test/ejabberd_tests/tests/muc_SUITE.erl#L3823